utils/cargo-tfhe-lints-inner/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "cargo-tfhe-lints-inner"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+rustc-tools = "0.78"
+
+[package.metadata.rust-analyzer]
+rustc_private=true
```

Comment:
just for my own knowledge what does this do ?

Reply:
The compiler internal API is accessed through "special" crates that are not present in the `Cargo.toml`. This line is used to make rust-analyzer aware of these crates: https://rust-analyzer.github.io/manual.html#:~:text=rust%2Danalyzer.rustc.source